### DISCUSSION TITLE
How do we get and set the default database fetchMode?

### DISCUSSION THREAD

**Discussion by drnkwati** [laravel/framework#44843] [Q&A]
Hi
The Illuminate\Database\Connection getFetchMode method has been removed. 
Any ideas how we get or set the default fetchMode?
thanks

**Comment by henzeb** [comment]
I didn't know either, and I decided to use google. 
https://mitrallex.medium.com/just-in-case-fetch-mode-for-pdo-connection-in-laravel-328fdbac6449

**Reply by drnkwati** [reply]
Thanks
I think removing the setFetchMode method was a bad decision because the property(fetchMode) still exist on the connection. My project integrates standalone Illuminate Database among other ORM like doctrine and the fetchMode needs to change during execution of queries.
